data_steward/analytics/cdr_ops/rt_cdr_qc/cdr_deid_qa_report4_dateshift.py

```diff
@@ -31,70 +32,81 @@
 deid_cdr = ""
 deid_questionnaire_response_map_dataset = ""
 pipeline=""
-run_as = ""
-# -
+run_as=""
 
-# df will have a summary in the end
-df = pd.DataFrame(columns = ['query', 'result']) 
+# +
+# Parameters
+#project_id = "aou-res-curation-prod"
```

Comment:
This parameter should be removed, we do not want our dataset and project names in the git history.